CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # PowerShell Editor Services Release History
 
+## v2.0.0-preview.6
+### Thursday, October 31, 2019
+
+#### Special Note
+In this release of the preview extension,
+we've merged significant architectural work into PowerShell Editor Services.
+After several months of work, PSES now uses the Omnisharp LSP library
+to handle Language Server Protocol interaction instead of rolling its own,
+allowing PSES to concentrate on being a good PowerShell backend.
+We hope you'll see increased performance and stability in this release.
+As always, [please let us know if you find any issues](https://github.com/PowerShell/vscode-powershell/issues/new/choose).
```

Comment:
Since this is editor services change log not sure if should link issues to this repo instead?

Reply:
Ah good point. Usually I'd say link to vscode, but yeah if you're reading this changelog, probably right!